guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -42,38 +73,68 @@ Catalogs are compiled by a Puppet server from Puppet Manifests and data from Pup
+You can create compliance policies using the {ProjectWebUI}.
+Compliance reports require the setting of a specific XCCDF profile from a SCAP content, optionally using a tailoring file.
+You can setup scheduled tasks on {ProjectServer} that check your hosts for compliance against SCAP content.
 
 [[Compute_profile]]
 Compute profile:: Specifies default attributes for new virtual machines on a compute resource.
 
 [[Compute_resource]]
-Compute resource:: A virtual or cloud infrastructure, which {ProjectName} uses for deployment of hosts and systems.
-Examples include {oVirt}, {OpenStack}, EC2, and VMWare.
+Compute resource::
+A compute resource is an external virtualization or cloud infrastructure that you can attach to {Project}.
+{Project} can provision, configure, and manage hosts within attached compute resources.
+Examples for compute resources include VMware or libvirt and cloud providers such as Microsoft Azure or Amazon EC2.
+You can access compute resources in the {ProjectWebUI} under *Infrastructure* > *Compute Resources*.
+
+[[Configuration_Management]]
+Configuration Management::
+Configuration management describes the task of configuring and maintaining hosts.
+In {Project}, you can use
+ifdef::satellite[]
+Ansible and Puppet
+endif::[]
+ifndef::satellite[]
+Ansible, Puppet, and Salt
+endif::[]
+to configure hosts and maintain everything in one place as a single source of infrastructure truth.
 
 [[Container]]
 Container (Docker container):: An isolated application sandbox that contains all runtime dependencies required by an application.
 {Project} supports container provisioning on a dedicated compute resource.
 
 [[Container_image]]
-Container image:: A static snapshot of the container’s configuration.
+Container image:: A static snapshot of the container's configuration.
 {Project} supports various methods of importing container images as well as distributing images to hosts through content views.
 
 [[Content]]
-Content:: A general term for everything {Project} distributes to hosts.
-Includes software packages (RPM files), or Docker images.
+Content::
+A general term for everything {Project} distributes to hosts.
+Content includes software packages such as `.{client-pkg-ext}` packages, errata, or Docker images.
 Content is synchronized into the Library and then promoted into lifecycle environments using content views so that they can be consumed by hosts.
 
 [[Content_delivery_network_CDN]]
 Content delivery network (CDN):: The mechanism used to deliver Red{nbsp}Hat content to {ProjectServer}.
 
 [[Content_host]]
-Content host:: The part of a host that manages tasks related to content and subscriptions.
+Content host::
+A content host receives content from your {ProjectServer} or {SmartProxyServers}.
+// TODO: still true after https://www.youtube.com/watch?v=XKuMf9Li734&t=446s ???
+Content hosts are subscribed to {Project} and associated to exactly one content view and exactly one lifecycle environment.
```

Comment:
Yes, but see https://github.com/theforeman/foreman-documentation/issues/2795

I still think we should drop using this term and just use Hosts.